aptos-move/block-executor/src/executor.rs

```diff
@@ -404,7 +421,9 @@ where
         last_input_output: &TxnLastInputOutput<T::Key, E::Output, E::Error>,
         versioned_cache: &MVHashMap<T::Key, T::Value, X>,
         scheduler: &Scheduler,
+        // TODO: should not need to pass base view.
         base_view: &S,
+        sync_view: &mut LatestView<T, S, X>,
         role: CommitRole,
```

Comment:
The `work_task_with_scope` function takes both `base_view` and `sync_view` as input? I thought you were renaming base_view with sync_view. Are they different?